doc/release-notes.md

```diff
@@ -25,18 +25,46 @@ is not possible to compute that without knowing the number of logical actions
 in a transaction, which was not an input to `estimatefee`. The `fee_estimates.dat`
 file is no longer used.
 
+RPC Changes
+-----------
+
+A new RPC method, `z_getsubtreesbyindex`, has been added to the RPC interface.
+This method is only enabled when running with the `-experimentalfeatures=1` and
+`-lightwalletd=1` node configuration options. This method makes available to the
+caller precomputed node values within the Sapling and Orchard note commitment 
+trees. Wallets can make use of these precomputed values to make their existing
+notes spendable without needing to fully scan the sub-trees whose roots
+correspond to the returned node values.
+
+In conjunction with this change, the `getblock` RPC method now returns an
+additional `trees` field as part of its result. The value for this field is an
+object that contains the final sizes of the Sapling and Orchard note commitment
+trees after the block's note commitments have been appended to their respective
+trees.
```

Comment:
Are these only present when `-experimentalfeatures=1` and `-lightwalletd=1` are set?

Reply:
I did not gate this on lightwalletd being enabled.

Reply:
I don't think there's a reason to condition this on lightwalletd info either. This is good for debugging even outside the lightwalletd context.